codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/customize/RequiredCustomizations.kt

```diff
@@ -85,7 +85,7 @@ class RequiredCustomizations : ClientCodegenDecorator {
             Feature(
                 "rt-tokio",
                 true,
-                listOf("aws-smithy-async/rt-tokio", "aws-smithy-types/rt-tokio"),
+                listOf("aws-smithy-async/rt-tokio", "aws-smithy-types/rt-tokio", "aws-smithy-types/http-body-1-x"),
```

Comment:
Does this belong here? Seems odd that we're enabling the `rt-tokio` feature and also `http-body-1-x` based on the surrounding code/context.